Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/EditorComponentModeBus.h

```diff
@@ -241,10 +241,16 @@ namespace AzToolsFramework
             using BusIdType = AzFramework::EntityContextId;
             static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::ById;
 
+            /// Called when a Component Mode is started.
+            virtual void ComponentModeStarted(const AZ::Uuid& /*componentType*/) {}
+
             /// Called when Tab is pressed to cycle the 'selected' ComponentMode (which shortcuts/actions are active).
             /// Also called when directly selecting a Component in the EntityOutliner.
             virtual void ActiveComponentModeChanged(const AZ::Uuid& /*componentType*/) {}
 
+            /// Called when Component Mode is ended.
+            virtual void ComponentModeEnded() {}
```

Comment:
update: Please remove these calls and use the other notification bus mentioned earlier

Reply:
Thanks!